graphql/schema/mutation.graphql

```diff
@@ -26,6 +26,16 @@ type Mutation {
     apiIssue: IssueLinkInput!
     isIssue: Boolean!
   ): Boolean!
+  addAnnotationMetadataLink(
+    taskId: String!
+    execution: Int!
+    apiMetadataLink: MetadataLinkInput!
+  ): Boolean!
+  removeAnnotationMetadataLink(
+    taskId: String!
+    execution: Int!
+    apiMetadataLink: MetadataLinkInput!
+  ): Boolean!
```

Comment:
I was also under this assumption, although I suppose there's really no reason not to have it be consistent with everything else in a task annotation and have it be editable on the failures detail tab by people with correct permissions, especially since the work is half done now.

Reply:
Yes my assumption was that both would be supported 🤷